test/test.js

```diff
@@ -1,5 +1,52 @@
+const path = require("path")
 const test = require("tape").test
+const FlyNotify = require("../")
+const defaults = {
+  title: "Fly",
+  message: "Task completed.",
+  icon: path.resolve(__dirname, "..", "fly-logo.png"),
+}
+const plugin = new FlyNotify()
 
-test("fly-notify", function (t) {
- t.end()
+// Adding this to make sure tests pass since we're not checking that the `node-
+// notifier` functionality works.
+plugin.error = function() {}
+plugin.debug = function() {}
+
+test("Testing `fly-notify`", function (t) {
+  t.plan(2)
+  t.ok(plugin.notify, "Fly-Notify `notify` method exists ~~%")
+  t.equal(typeof plugin.notify, "function", "Fly-Notify `notify` method is a function ~~%")
+  t.end()
+})
+
+test("Testing bad options", function(t) {
```

Comment:
Yes. Adding a lot of tests does not make your app more safe. Adding the right tests does.

See the tests for `fly-babel`:

```js
const test = require("tape").test
const bind = require("fly-util").bind
const babel = require(bind("../"))

test("fly-babel", (t) => {
  t.plan(2)
  babel.call({
    filter: function (name, transform) {
      t.equal(name, "babel", "add babel filter")
      t.ok(/var a/.test(transform("let a = 0")), "babel transform")
    }
  })
})
```

This is almost enough. Also need to check it is returning the correct extension. But other than that, that's all the plugin should test for. Fly already tests the behavior `Fly.proto.filter`.